src/searchForm/SearchForm.tsx

```diff
@@ -144,9 +143,11 @@ const SearchForm = ({
             className="kirk-search-button"
             onClick={() => setElementOpened(Elements.AUTOCOMPLETE_FROM)}
           >
-            <Bullet type={BulletTypes.SEARCH} className="kirk-bullet--searchForm" />{' '}
+            <span className="kirk-bullet--searchForm">
+              <Bullet type={BulletTypes.SEARCH} />
+            </span>
             <TextBody>
-              {formValues[Elements.AUTOCOMPLETE_FROM] || autocompleteFromProps.placeholder}
+              {autocompleteFromValue?.item.label || autocompleteFrom.props.placeholder}
```

Comment:
I need to check if this is polyfilled

Reply:
As we don't use babel on kirk, i'm not sure. But since we build the app in kairos with babel, I don't think it'll be a problem for us.

Reply:
Checked, working in SPA on IE 11 👌